models.py

```diff
@@ -0,0 +1,11 @@
+from flask_sqlalchemy import SQLAlchemy
+
+
+db = SQLAlchemy()
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(40), nullable=False)
+    surname = db.Column(db.String(80), nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(120), nullable=False)
```

Comment:
У вас пароль всегда будет длиной 64 символа. Значит размер пол можно уменьшить
```suggestion
    password = db.Column(db.String(64), nullable=False)
```